lib/inputs/select-input.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+var _ = require('underscore');
+var BaseInput = require('./base-input');
+var errors = require('../errors');
+
+class SelectInput extends BaseInput {
+    constructor(id, params) {
+        super('select', id, params);
+    }
+
+    getInputDefaultValue() {
+        return this.params.items[0].value;
+    }
+
+    normalizeParams(params) {
+        params = Object.assign({}, params);
+
+        if (!params.items) {
+            throw errors.compileError('INPUT-REQ-PARAMS-MISSING', {
+                input_id: this.id,
+                missing_params: 'items'
+            });
+        }
+
+        if (!_.isArray(params.items)) {
+            throw errors.compileError('INPUT-INVALID-PARAM', {
+                input_id: this.id,
+                param: 'items',
+                param_value: params.items,
+                message: 'Item param must be an array'
```

Comment:
"Item" doesn't need to be in `message` because its already in `param` above, right?

If it is there, it should be lowercased and quoted.